contracts/Paper.sol

```diff
@@ -51,13 +58,43 @@ contract Paper {
         require(block.timestamp <= deadlineDate, "The deadline date has passed.");
         _;
     }
+
+    modifier primaryCheckedHasPassed() {
+        require(primaryChecked == true);
+        _;
+    }
+
+    function primaryChecking() public onlyOwner() {
+        primaryChecked = true;
+    }
     
-    function addFeedback(string memory _feedback) public canBeReview() {
-        feedbacks.push(FeedBack(_feedback, msg.sender));
+    function addFeedback(string memory _feedback) public  primaryCheckedHasPassed() canBeReview() {
```

Comment:
Rather than adding multiple modifiers, you should use a single one taking the expected state.

- [ ]  Define what each state of the process are in the `State` enum, and what can be done during them
- [ ] Create a modifier `isState` which takes the expected state as parameter and asserts that it is the current state
- [ ] Add this modifier to each public function